Fitness/Views/GymFacilitiesView.swift

```diff
@@ -0,0 +1,483 @@
+            cell.dayLabel.text = facilityData[0].data[indexPath.row].1
+            
+            return cell
+        default:
+            return UITableViewCell()
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        guard let unwrappedFacilityName = self.facilityName else {
+            return nil
+        }
+        
+        switch unwrappedFacilityName {
+        case .helenNewman:
+            let header = tableView.dequeueReusableHeaderFooterView(withIdentifier: FacilityHoursHeaderView.identifier) as! FacilityHoursHeaderView
+            var gesture: FacilitiesGestureRecognizer!
+            
+            if tableView == basketballTableView {
+                header.iconImageView.image = UIImage(named: "basketball")
+                header.facilityNameLabel.text = facilityData[0].facility
+                gesture = FacilitiesGestureRecognizer(target: self, action: #selector(self.dropFacilityHours(sender:) ))
+                gesture.title = "basketball"
+
+            } else if tableView == bowlingTableView {
+                header.iconImageView.image = UIImage(named: "bowling")
+                header.facilityNameLabel.text = facilityData[1].facility
+                gesture = FacilitiesGestureRecognizer(target: self, action: #selector(self.dropFacilityHours(sender:) ))
+                gesture.title = "bowling"
+            } else {
+                header.iconImageView.image = UIImage(named: "pool")
+                header.facilityNameLabel.text = facilityData[2].facility
+                gesture = FacilitiesGestureRecognizer(target: self, action: #selector(self.dropFacilityHours(sender:) ))
+                gesture.title = "pool"
+            }
+            
+            header.addGestureRecognizer(gesture)
+            
+            return header
+        case .teagle:
+            let header = tableView.dequeueReusableHeaderFooterView(withIdentifier: FacilityHoursHeaderView.identifier) as! FacilityHoursHeaderView
+            header.iconImageView.image = UIImage(named: "pool")
+            header.facilityNameLabel.text = facilityData[0].facility
+            let gesture = FacilitiesGestureRecognizer(target: self, action: #selector(self.dropFacilityHours(sender:) ))
+            header.addGestureRecognizer(gesture)
+            
+            return header
+        default:
+            return nil
+        }
+    }
+        
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 28
+    }
+    
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        guard let unwrappedFacilityName = self.facilityName else {
+            return 0
+        }
+        
+        switch unwrappedFacilityName {
+        case .helenNewman:
+            if tableView == basketballTableView! {
+                return facilityData[0].isDropped ? 36 : 36
```

Comment:
Any reason not to just `return 36` here or will we have varying header sizes in the future?